### PR TITLE
(docs) Updated guidelines with choco version requirement

### PR DESCRIPTION
**Pull request by AdmiringWorm**
Just thought it would be nice to have something mentioning which version of chocolatey we should support, I'm suggesting with this PR to support all versions of chocolatey that was made available in the last 12 months (unless a dependency on chocolatey is added), I also talked this over with @gep13 which seemed to agree.

Opening this in the form of discussion though, as I would love different opinions on the matter

/cc @majkinetor @pascalberger

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -109,6 +109,11 @@ Packages should have uninstaller if auto uninstall doesn't work.
 
 Chocolatey supports PowerShell v2+ because it supports Windows 7+/Windows Server 2008+ (Windows Server 2003 is supported currently, but is deprecated and expected to be removed in a future version). Due to that, PowerShell v2 is expected to work when you run anything here.
 
+### 1.1.13 Chocolatey compatibility
+
+Packages is expected to be compatible with the chocolatey version available 12 months ago.
```
Comment (gep13) [inline CONTRIBUTING.md]: Minor nit picks: 

`is` -> `are`
`chocolatey` -> `Chocolatey`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -109,6 +109,11 @@ Packages should have uninstaller if auto uninstall doesn't work.
...
+If a later version of chocolatey is required, then a dependency on the earliest version compatible with the package must be added.
```
Comment (gep13) [inline CONTRIBUTING.md]: Minor nit pick:

`chocolatey` -> `Chocolatey`

File: README.md [hunk 3]
```diff
@@ -16,13 +16,14 @@ This repository contains chocolatey packages, most of which are [automatic](http
 These packages are managed and maintained by the Chocolatey community package maintainers core team, and also supported by you (the community)! To get a package added here, the official [Chocolatey Account](http://chocolatey.org/profiles/chocolatey) has to have push access to the package on [chocolatey.org](http://chocolatey.org).
 
 **All packages in this repo should be in conformity with the [contributing guidelines](CONTRIBUTING.md).** Contributions that follow the guideliness are welcome.
+All packages in this repo is expected to be installable with any version of chocolatey being made available in the past 12 months, unless a dependency on chocolatey has been added.
```
Comment (gep13) [inline README.md]: Minor nit picks:

`is` -> `are`
`chocolatey` -> `Chocolatey`

File: CONTRIBUTING.md (hunk 1 above)
Comment (AdmiringWorm) [reply CONTRIBUTING.md]: I can never remember when to use `is` vs `are`.
Thanks, I'll get those fixed

**Comment by AdmiringWorm** [comment]
I'll leave this open until sunday, if no-one have any objections until then (or corrections) I'll merge it.

**Pull request merged**